ci/pipeline.yml

```diff
@@ -41,6 +41,16 @@ test: &test
 
 jobs:
 
+  - name: set-pipeline-((deploy-env))
+    serial: true
+    plan:
+      - get: src
+        resource: src-((deploy-env))
+        params: {depth: 1}
+        trigger: true
+      - set_pipeline: build-container
+        file: src/ci/pipeline.yml
```

Comment:
Do we need to add instance variables here for staging and prod?

Reply:
Added!